querydsl-core/src/test/java/com/querydsl/core/serialization/SerializerBaseTest.java

```diff
@@ -43,3 +42,26 @@ public void test() {
     }
 
 }
+
+class DummySerializer extends SerializerBase<DummySerializer> {
```

Comment:
When I make a simple concrete class to test an abstract class, I tend to bundle it with the test so that its purpose is clear, but you have a good point so I'll move it back when I get home.